client/extensions/woocommerce/state/data-layer/ui/products/test/index.js

```diff
@@ -244,8 +244,9 @@ describe( 'handlers', () => {
 			};
 		} );
 
-		test( 'should return null when there are no edits', () => {
-			expect( makeProductActionList( null, 123, undefined ) ).to.equal.null;
+		test.skip( 'should return null when there are no edits', () => {
```

Comment:
@coderkevin looks like this test never really worked.. should we fix it? remove it?

Reply:
Hmm, I don't remember having failing tests long-term. I can look into this tomorrow.

Reply:
@coderkevin it wasn't failing before because the assertion itself was invalid :)

Reply:
in previous versions of chai, using broken property names didn't throw an error. So `.to.equal.nonsense` would pass just fine. `.null` is not a valid property name, so it would not throw, and consequently pass, in the old version of chai. 

The new version uses Proxies and can (finally) catch these kinds of typos.

Reply:
@coderkevin I'm going to land this as is (with the skip). If / when you get some time to address this, feel free to ping me on the review.